src/AuthenticationController.js

```diff
@@ -114,6 +114,13 @@ class Authentication extends AbstractController {
     return passport.authenticate('local')
   }
 
+  get logoutMiddleware () {
+    return (req, res, next) => {
+      req.logout()
```

Comment:
Does `req.logout()` ever throw? (i.e. if you aren't logged in)

Reply:
No, it just removes `req.user`.
From the docs (http://www.passportjs.org/docs/logout/): 
> Invoking logout() will remove the req.user property and clear the login session (if any).

Reply:
Nice. I'm happy then.